torch/_tensor_docs.py

```diff
@@ -3495,6 +3495,13 @@ def callable(a, b) -> number
 See :func:`torch.trunc`
 """)
 
+add_docstr_all('fix',
+               r"""
+trunc() -> Tensor
+
+Alias for :meth:`~Tensor.trunc`.
```

Comment:
Let's keep the alias info in the function documentation and have this point to :func:`torch.fix` (and have the inplace version say it's an inplace version of this).